sechub-cli/src/mercedes-benz.com/sechub/cli/config_test.go

```diff
@@ -160,3 +163,54 @@ func Example_normalizeCMDLineArgs() {
 	// [./sechub -configfile my-sechub.json -wait 30 scan]
 	// [./sechub -version]
 }
+
+func Example_tempFile() {
```

Comment:
Those are actually two test cases. One config is configured to use "." the other "/tmp/my_dir". I would prefer two functions for that. For each test case its own function. Something like:

~~~
Example_tempFile_current_dir() {}
Example_tempFile_tmp_dir() {}
~~~

This would be more in tune with the way we write our test cases in the Java part. In addition, it is much easier to read and understand if it would be two cases.

Reply:
Okay. Split into two separate tests.